backend/src/main/java/com/ody/common/config/WebConfig.java

```diff
@@ -27,4 +33,11 @@ public void addArgumentResolvers(List<HandlerMethodArgumentResolver> resolvers)
     public void addInterceptors(InterceptorRegistry registry) {
         registry.addInterceptor(loggingInterceptor);
     }
+
+    @Override
+    public void addCorsMappings(CorsRegistry registry) {
+        registry.addMapping("/admin/api-call/**")
+                .allowedOrigins(allowedOrigins)
+                .allowedMethods("GET", "POST");
```

Comment:
전역적인 CORS 설정에 대한 이해도가 부족해서 reference를 찾아보았는데 maxAge를 통해 preflight request에 대한 캐싱 유효기간을 설정할 수 있더라구요. 반영해보아도 좋을 것 같아요.

```suggestion
                .allowedMethods("GET", "POST")
                .maxAge(3000); 원하는 시간 만큼 pre-flight 리퀘스트 캐싱
```

Reply:
또한 admin controller를 따로 분리하는 것으로 알고 있는데 webConfig가 아니라 Controller 단에서만 CORS를 설정하는 방법도 있어서 공유합니다! 반영은 제리가 자유롭게 해주세요!

```java
@Controller
@CrossOrigin(origins = "*", methods = RequestMethod.GET) 
public class customController {

    @GetMapping("/url")  
    @ResponseBody
    public List<Object> findAll(){
        return service.getAll();
    }
}
```

Reply:
컨트롤러에서 cors 설정하는 것도 고려했는데 설정 로직들이 한 곳에 모여있는 것과 설정이 적용되는 곳에서 바로 설정을 확인할 수 있는 것 중 어떤 것이 더 나은가요?

Reply:
preflight는 무작정 요청을 보냈다가 cors 위반 요청이라면 리소스 낭비 및 서버 부하에 영향을 주기 때문에 
요청이 유효한지 options 메서드를 이용해 예비 요청을 보내기 것인데요

현재(그리고 앞으로도) 어드민 요청이 get 요청, body 없는 post 요청이라 실패하더라도 부담이 없고 허용 메서드에도 options를 포함하고 있지 않아서 preflight를 설정하지는 않았습니다

그렇지만 저도 새롭게 preflight request를 배웠네요 감사합니다 👍👍